File: test/fixtures/output/node/fetch/custom-method.js
const fetch = require('node-fetch');

const url = 'https://httpbin.org/anything';
const options = {method: 'PROPFIND'};

fetch(url, options)
  .then(res => res.json())
  .then(json => console.log(json))
  .catch(err => console.error('error:' + err));

File: src/targets/go/native/fixtures/custom-method.go
package main

import (
	"fmt"
	"net/http"
	"io"
)

func main() {

	url := "https://httpbin.org/anything"

	req, _ := http.NewRequest("PROPFIND", url, nil)

	res, _ := http.DefaultClient.Do(req)

	defer res.Body.Close()
	body, _ := io.ReadAll(res.Body)

	fmt.Println(res)
	fmt.Println(string(body))

}